### ISSUE TITLE
Update Gimlet app to the latest version

### ISSUE THREAD

**Issue by aakashnagpal4567** [civo/kubernetes-marketplace#478]
This issue is a:
* [ ] Bug Report
* [x] Enhancement suggestion
* [ ] Feature request

If the issue concerns an existing Marketplace application, please tag the maintainer of the application (found in `manifest.yaml`) below:

## Enhancement suggestions / feature requests

Currently, we have Gimlet version v0.4.2 installed by Civo marketplace as a newer version v0.23.2 is available, so it needs to be updated to the newer version. We kindly request an update to the latest version of Gimlet on the Civo marketplace to ensure that we have access to the most recent features and improvements.

@laszlocph is the maintainer for this app, so I request you to please take a look into this.

**Comment by laszlocph** [comment]
Good point @aakashnagpal4567 please allow me a couple of weeks to do this.
We are moving to a simplified installation experience with Helm, landing in about 2 weeks.

**Comment by laszlocph** [comment]
https://github.com/civo/kubernetes-marketplace/pull/542

**Comment by olblak** [comment]
@laszlocph If you still want to rely on pinned version with pullrequest validation, several months ago we started an experimentation to automate version update.

You would need to drop an update manifest in `.github/updatecli/updatecli.d` similar to https://github.com/civo/kubernetes-marketplace/blob/master/.github/updatecli/updatecli.d/rancher.yaml 

Which automatically open a pullrequest such as https://github.com/civo/kubernetes-marketplace/pull/455

Updatecli is executed every hour by this workflow https://github.com/civo/kubernetes-marketplace/blob/master/.github/workflows/updatecli.yml

Feel free to ping if you need help

**Comment by laszlocph** [comment]
@olblak this is neat.

However the installation method does not need a version anymore. I included `version: v0.17.2` assuming CIVO needs it.
Although I could update that with updatecli. Or skip the version completely. Could you tell @aakashnagpal4567 if you need the version field in the yaml?
